js/templates/pageNav.html

```diff
@@ -32,20 +32,25 @@
       </div>
       <div class="rowDivV colBorderBackground"></div>
       <div>
-        <div class="flexVCent">
+        <div class="flexVCent box">
           <a class="iconBtn ion-android-notifications"></a>
-          <a class="discSm border colBorder"
+          <a class="discSm border colBorder js-navListBtn"
           <% if (ob.navimage) { %>
           //this will be updated to the real image parameter
           style="background-image: url(ob.navImage), url('../imgs/defaultUser.png')"
           <% } else { %>
             style="background-image: url('../imgs/defaultUser.png')"
           <% } %>
           ></a>
+          <nav class="box border boxList navList colBorder colPrimary padKids shad2 foldDown js-navList">
```

Comment:
In this case, this is a dummy list just so we can test out going to the page and making the open and close work. 

Once we get styles from Mike, we'll give it specific styling, it'll probably only have a few of the base classes.